src/tests/EdFi.DataManagementService.Tests.E2E/Features/Descriptors/CreateDescriptorsValidation.feature

```diff
@@ -162,112 +157,109 @@ Feature: Descriptors are a set of mechanisms to support flexible enumerations or
                   """
                     {
                         "codeValue": "                      ",
-                        "description": "                    ",
-                        "namespace": "uri://ed-fi.org/AbsenceEventCategoryDescriptor",
+                        "namespace": "  ",
                         "shortDescription": "                    "
                     }
                   """
              Then it should respond with 400
               And the response body is
-                  """{
+                  """
+                  {
                     "validationErrors": {
                         "$.codeValue": [
-                            "codeValue cannot contain leading or trailing spaces."
+                        "codeValue cannot contain leading or trailing spaces."
                         ],
                         "$.namespace": [
-                            "namespace cannot contain leading or trailing spaces."
-                            ],
+                        "namespace cannot contain leading or trailing spaces."
+                        ],
                         "$.shortDescription": [
-                            "shortDescription cannot contain leading or trailing spaces."
+                        "shortDescription cannot contain leading or trailing spaces."
                         ]
                     },
                     "errors": [],
                     "detail": "Data validation failed. See 'validationErrors' for details.",
-                    "type": "urn:ed-fi:api:bad-request:data",
+                    "type": "urn:ed-fi:api:bad-request:data-validation-failed",
                     "title": "Data Validation Failed",
                     "status": 400,
                     "correlationId": null
                   }
                   """
 
-        # Descriptors are not validating properly. DMS-295
-        @ignore
         Scenario: 07 Ensure clients cannot create a descriptor with leading spaces in required attributes
              When a POST request is made to "/ed-fi/absenceEventCategoryDescriptors" with
                   """
                     {
                         "codeValue": "                      a",
                         "description": "                    a",
-                        "namespace": "uri://ed-fi.org/AbsenceEventCategoryDescriptor",
+                        "namespace": "   uri://ed-fi.org/AbsenceEventCategoryDescriptor",
```

Comment:
As before, `namespace` is one of the required attributes, therefore it too needs the leading spaces. Same with the next scenario, where we have trailing spaces.